testing/requirements.txt

```diff
@@ -13,7 +13,8 @@ zbar-py==1.0.4
 
 # NFC and NDEF handling
 nfcpy==1.0.3
-ndef==0.2
+# commentt out below dependency 'ndef==0.2' if on debian based OS
+#ndef==0.2
```

Comment:
not sure if this is needed in MacOS. But when I install nfc.py it also pulls as a dependency ndeflib which is what we want. ndef==0.2 was obsolete and does not work

Reply:
`ndeflib` in install requirements of nfcpy so okay to remove; kill comment too imho